tests/suites/os/tests/iptables/index.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+module.exports = {
+	title: 'iptables rules tests',
+	tests: [
+		{
+			title: 'BALENA-FIREWALL test',
+			run: async function(test) {
+				let check = await this.worker.executeCommandInHostOS(
+                    `iptables -L -vn -t filter | grep "Chain BALENA-FIREWALL" && echo $?`,
```

Comment:
Even better would be to verify that their are rules for the BALENA-FIREWALL chain based on number of lines. For example:

```
root@f81df08:~# iptables -t filter -L BALENA-FIREWALL -vn |wc -l
17
```